doc/source/examples.rst

```diff
@@ -92,42 +92,80 @@ project.
 Systemd Container
 =================
 
-The docker daemon was designed to provide a simple means of starting, stopping
-and managing containers. It was not originally designed to bring up an entire
-Linux system or manage services for such things as start-up order, dependency
-checking, and failed service recovery. [1]_
+To start a service which requires systemd, in an `unprivileged` container,
+configure `molecule.yml` with a systemd compliant image, tmpfs, volumes,
+and command as follows. [1]_
```

Comment:
Hmmm, sorry to nitpick now but no other part of the documentation is using these `[N]` links ...

Would you mind fixing to use the typical RST links like:

```
`my link`_

.. _my link: URL
```